### ISSUE TITLE
Infeasible problems cause R to crash

### ISSUE THREAD

**Issue by travis-leith** [goulart-paul/osqp-r#29]
R version 4.2.1 (2022-06-23 ucrt) -- "Funny-Looking Kid"
Package osqp version 0.6.0.5

When I run the following infeasible problem, R crashes.

```R
rm(list = ls())

library(osqp)
## example, adapted from OSQP documentation
library(Matrix)

P <- Matrix(c(11., 0.,
              0., 0.), 2, 2, sparse = TRUE)
q <- c(3., 4.)
A <- Matrix(c(-1., 0., -1., 2., 3.,
              0., -1., -3., 5., 4.)
            , 5, 2, sparse = TRUE)
u <- c(0., 0., -15., 100., 80)
l <- u + 1

settings <- osqpSettings(verbose = TRUE)

# Solve with OSQP
res <- solve_osqp(P, q, A, l, u, settings)
res$x
```

**Comment by bnaras** [comment]
Ah, I do confirm this on osqp 0.6.0.7 as well.  (My initial reply was wrong.)

**Comment by travis-leith** [comment]
This problem still exists with R4.2.2. Would be awesome if we could get it fixed.

**Comment by travis-leith** [comment]
problem still exists with R4.3.0

If I had the skills to fix this, would love to. Sadly, I do not know enough about c++

**Comment by bnaras** [comment]
I'm closing this issue as addressed in version 0.6.3 with a check at the R side. So an error will be thrown rather than a crash.
